libsndfile.yaml

```diff
@@ -0,0 +1,78 @@
+# Generated from https://git.alpinelinux.org/aports/plain/main/libsndfile/APKBUILD
+package:
+  name: libsndfile
+  version: 1.2.0
+  epoch: 0
+  description: C library for reading and writing files containing sampled sound
+  copyright:
+    - license: LGPL-2.1-or-later
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - build-base
+      - automake
+      - autoconf
+      - opus-dev
+      - opus
+      - alsa-lib-dev
+      - cmake
+      - flac-dev
+      - libogg-dev
+      - libvorbis-dev
+      - speexdsp-dev
```

Comment:
libspeex and libspeexdsp are different, one is for loading speex files (which I know libsndfile has a backend for), while the other is for voice chat mostly.